### DISCUSSION TITLE
Reboot in loop to MacOS base system on Qemu

### DISCUSSION THREAD

**Discussion by Treza88** [sickcodes/Docker-OSX#528] [Q&A]
Hello,
I have a problem to istall MacOS monterey on windows 11 with dokcker, i used this:
https://hub.docker.com/r/sickcodes/docker-osx#id-like-to-run-docker-osx-on-windows

But, when i run Macos base system, I arrive on disk utility or further away on reinstall MacOS and it reboot in loop to MacOS base system indefinitely

Whats the problem?

For more explanation, ask me

Sorry for my English, i'm French

**Comment by evilz** [comment]
Same here, I also tried the prebuild Catalina and update but still not working. 
Any ideas?
